services/QuillLMS/app/controllers/concerns/growth_results_summary.rb

```diff
@@ -0,0 +1,151 @@
+module GrowthResultsSummary
+  extend ActiveSupport::Concern
+
+  NOT_PRESENT = 'Not present'
+  NOT_CORRECT = 'Not correct'
+  FULLY_CORRECT = 'Fully correct'
+  PARTIALLY_CORRECT = 'Partially correct'
+
+  NO_PROFICIENCY = 'No proficiency'
+  PARTIAL_PROFICIENCY = 'Partial proficiency'
+  PROFICIENCY = 'Proficiency'
+  GAINED_PROFICIENCY = 'Gained proficiency'
+  MAINTAINED_PROFICIENCY = 'Maintained proficiency'
+
+  extend self
+
+  def growth_results_summary(current_user, pre_test_activity_id, post_test_activity_id, classroom_id)
+    @current_user = current_user
+    pre_test = Activity.find(pre_test_activity_id)
+    @skill_groups = pre_test.skill_groups
+    set_pre_test_activity_sessions_and_assigned_students(pre_test_activity_id, classroom_id)
+    set_post_test_activity_sessions_and_assigned_students(post_test_activity_id, classroom_id)
+    @skill_group_summaries = @skill_groups.map do |skill_group|
+      {
+        name: skill_group.name,
+        description: skill_group.description,
+        not_yet_proficient_in_pre_test_student_names: [],
+        not_yet_proficient_in_post_test_student_names: []
+      }
+    end
+
+    {
+      student_results: student_results,
+      skill_group_summaries: @skill_group_summaries
+    }
+  end
+
+  private def set_pre_test_activity_sessions_and_assigned_students(activity_id, classroom_id)
+    unit_ids = @current_user.units.joins("JOIN unit_activities ON unit_activities.activity_id = #{activity_id}")
```

Comment:
Eek, thanks for calling this out. My limited local dataset obviously didn't reveal that issue and it didn't occur to me! These methods have undergone some change since this PR but this hasn't been corrected, so I'm going to make that change to the last PR for this project to minimize merging weirdness if that's okay with you.